stdlib/sql/to_privates_test.go

```diff
@@ -0,0 +1,252 @@
+package sql
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	flux "github.com/influxdata/flux"
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/internal/errors"
+)
+
+// additional and seperate tests that can be run without needing functions to be Exported in sql, just to be testable
+func TestCorrectBatchSize(t *testing.T) {
+	// given the combination of row width and supplied batchSize argument from user, verify that it is modified as required
+	userBatchSize := 1000
+	rowWidth := 10
+	correctedSize := correctBatchSize(userBatchSize, rowWidth)
+	if !cmp.Equal(99, correctedSize) {
```

Comment:
I am confused by this result.

Reply:
The calculation is correct but it is not clear why - perhaps it should link to the explanation (which will need to be updated) in the code about batchsize?